test/commands/hello/org.test.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@salesforce/command/lib/test';
+import { ensureJsonMap, ensureString } from '@salesforce/ts-types';
+
+describe('hello:org', () => {
+    test.withOrg({ username: 'test@org.com' }, true)
+        .withConnectionRequest((request) => {
+            const requestMap = ensureJsonMap(request);
+            if (ensureString(requestMap.url).match(/Organization/)) {
+                return Promise.resolve({
+                    records: [
+                        {
+                            Name: 'Super Awesome Org',
+                            TrialExpirationDate: '2018-03-20T23:24:11.000+0000'
+                        }
+                    ]
+                });
+            }
+            return Promise.resolve({ records: [] });
+        })
+        .stdout()
+        .command(['hello:org', '--targetusername', 'test@org.com'])
+        .it('runs hello:org --targetusername test@org.com', (ctx) => {
+            expect(ctx.stdout).to.contain(
+                'Hello world! This is org: Super Awesome Org and I will be around until Tue Mar 20 2018!'
+            );
+        });
+});
```

Comment:
Is this providing sufficient test coverage of the source code?

Reply:
There is some issue with sfdx plugin and we are not able to run testcases. Once it's resolved will add additional test cases.